portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,18 +32,25 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> messages;
   private Comments c;
 
   @Override
   public void init() {
-    messages = new ArrayList<>();
     c = new Comments();
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Comment");
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity entity : results.asIterable()) {
+      String username = (String) entity.getProperty("userName");
+      String comment =  (String) entity.getProperty("comment");
```

Comment:
Nit: I generally don't try to "vertically align" code because it tends to make diffs larger when you modify the code later on. In this case, I'd just remove the extra space in `=  (String)` and not worry that the lines are one character off from each other.

Reply:
:+1: